extractor/src/main/java/org/schabi/newpipe/extractor/search/SearchInfo.java

```diff
@@ -70,4 +76,16 @@ public String getSearchString() {
     public String getSearchSuggestion() {
         return searchSuggestion;
     }
+
+    public boolean isCorrectedSearch() {
+        return this.isCorrectedSearch;
+    }
+
+    public void setCorrectedSearch(boolean correctedSearch) {
+        isCorrectedSearch = correctedSearch;
```

Comment:
```suggestion
        this.isCorrectedSearch = correctedSearch;
```
Use `this` here too, for consistency's sake

Reply:
For even more consistency, change that `correctedSearch` to `isCorrectedSearch`